storage/doc.go

```diff
@@ -331,6 +331,32 @@ to add a [custom audit logging] header:
 	// Use client as usual with the context and the additional headers will be sent.
 	client.Bucket("my-bucket").Attrs(ctx)
 
+# Experimental gRPC API
+
+This package includes support for the Cloud Storage gRPC API, which is currently
+in preview. This implementation uses gRPC rather than the current JSON & XML
```

Comment:
I'm not familiar with go's conventions, but if `in preview` is sufficient for "if we identify something before GA we have to break, you are expected to update and accommodate the break", this is good. If not, a stronger disclaimer might be warranted.

Reply:
I can add an additional sentence specifically stating this.